scripts/seed/datasets/integrated/generate.py

```
@@ -0,0 +1,324 @@
+    for playlist_id in range(1, count + 1):
+        created_at = random_datetime_between(BASE_DATE, NOW)
+        updated_at = random_datetime_between(created_at, NOW)
+        deleted = 1 if random.random() < 0.05 else 0
+        deleted_at = random_datetime_between(updated_at, NOW) if deleted else None
+        yield [playlist_id, random.randint(1, user_count), f"Playlist {playlist_id}", f"Seed playlist description {playlist_id}", deleted, format_dt(updated_at), format_dt(deleted_at), format_dt(created_at)]
+
+def generate_playlist_tracks(count: int, playlist_count: int, track_count: int):
+    used_pairs: set[tuple[int, int]] = set()
+    position_by_playlist: dict[int, int] = {}
+    for playlist_track_id in range(1, count + 1):
+        playlist_id = random.randint(1, playlist_count)
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)